test.properties

```diff
@@ -870,6 +870,16 @@
     operating.system.windowsserver.version=2012r2
     #operating.system.windowsserver.version=2016
 
+##
+## Partner Portal Salesforce
+##
+
+    site.initializer.partner.portal.login.url=https://test.salesforce.com
+    site.initializer.partner.portal.consumer.key=3MVG9.hNadsS7GK6GmwkauP3.KEFV0QK5e0ZrR1dnGRcAaDJgZJRUyX2wu3h96vp3B7PQFEKwN5QXusVAFUTz
+    site.initializer.partner.portal.consumer.secret=816A8DBCF7658F484891DC492466FE40E2AEEB488DE8CA3793D1F979F3F74E96
+    site.initializer.partner.portal.username=jair.medeiros@liferay.com.prm
+    site.initializer.partner.portal.password=U*V!q6hd7DX@FrR6Dfxo
+
```

Comment:
Hi @MGabriellaS, it shouldn't be sent with your test. It's a change that remains only locally.